flow/e2e/test_utils.go

```diff
@@ -100,6 +100,41 @@ func GetPgRows(conn *connpostgres.PostgresConnector, suffix string, table string
 	)
 }
 
+func checkIfMirrorEntriesRemoved(conn *connpostgres.PostgresConnector, flowJobName string) (bool, error) {
```

Comment:
I expect failures here due to delays in context cancellation, not sure if we should assert for it